tchannel/thrift/client.py

```diff
@@ -38,6 +40,11 @@
 )
 
 
+@deprecated(
+    "client_for is deprecated and will be removed in 0.19.0, " +
+    "please switch usage to tchannel.thrift.load in conjunction " +
+    "with tchannel.TChannel or tchannel.sync.TChannel."
```

Comment:
You can use lexical string concatenation. Instead of,

    ("foo" +
     "bar")

You can do

    ("foo"
     "bar")